modxcms/revolution
# Bug 16212

### What does it do?
Removes the forced mgr context for the findPolicy() function.

### Why is it needed?
See bug #16212

### How to test
1. Turn on info level logging.
2. Turn off the access_resource_group_enabled setting.
3. Access a page in the web context as an anonymous user.
4. No log messages about the mgr context should show up.

### Related issue(s)/PR(s)
Resolves #16212
